### DISCUSSION TITLE
How to get mesh name from a pre-compiled environment

### DISCUSSION THREAD

**Discussion by ganlumomo** [microsoft/AirSim#4360] [General]
Hi, I am currently working on a pre-compiled environments AirSimNH. I want to get a list of mesh name and their corresponding object ID. I have gone through the issues and discussion, however, I still don't have a clear idea how to do it. Any help is highly appreciated!

**Comment by rajat2004** [comment]
`simListAssets` might be helpful, along with `simGetSegmentationObjectID`. Can't confirm for sure it'll work though
